README.md

```diff
@@ -18,10 +37,13 @@ imperative you follow the guidance for Pull Requests and requisite testing.
 
    Numbered CICE releases since version 6 with associated documentation and DOIs. 
    
-* **Consortium Community Bulletin Board**: https://xenforo.cgd.ucar.edu/cesm/forums/cice-consortium.146/
+* **Consortium Community Forum**: https://xenforo.cgd.ucar.edu/cesm/forums/cice-consortium.146/
 
    First point of contact for discussing model development including bugs, diagnostics, and future directions.   
 
 * **Resource Index**: https://github.com/CICE-Consortium/About-Us/wiki/Resource-Index
 
    List of resources for information about the Consortium and its repositories as well as model documentation, testing, and development.
+
+## License
+See our [License](Licese.pdf) and [Distribution Policy](DistributionPolicy.pdf).
```

Comment:
This "License" link was broken when I looked at your rendered version.

Reply:
fixed